py-polars/polars/internals/lazy_functions.py

```diff
@@ -1305,13 +1306,15 @@ def cumsum(
     return cumfold(lit(0).cast(UInt32), lambda a, b: a + b, column).alias("cumsum")
 
 
-def spearman_rank_corr(
-    a: str | pli.Expr, b: str | pli.Expr, ddof: int = 1, propagate_nans: bool = False
+def corr(
```

Comment:
We need to keep the `spearman_rank_corr` and `pearson_corr` methods, but raise a deprecation warning. Right now, users code will erro.

Reply:
The original methods are back with deprecation warnings.